src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -219,7 +270,7 @@ const styles = StyleSheet.create({
   footerWrapper: (h) => ({height: h}),
   contentReaction: (heightReaction) => ({
     maxHeight: heightReaction,
-    marginBottom: heightReaction <= 0 ? tabBarHeight + 10 : 0
+    marginBottom: heightReaction <= 0 ? tabBarHeight + normalize(10) : 0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `normalize` function has been used to normalize the `tabBarHeight` value. This should ensure that the tab bar height is consistent across different screen sizes.
```diff
-    marginBottom: heightReaction <= 0 ? tabBarHeight + 10 : 0
+    marginBottom: heightReaction <= 0 ? tabBarHeight + normalize(10) : 0
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->